app/app.go

```diff
@@ -127,6 +127,23 @@ func Run(ctx context.Context, conf Config) (err error) {
 		return err
 	}
 
+	log.Info(ctx, "Charon run config",
```

Comment:
Went with single log because multi line looked very verbose. Also this can go out of screen, in that case we can use ctrl + F or cmd + F to find the value of desired flag.

Reply:
![Screenshot 2022-10-11 at 1 04 10 PM](https://user-images.githubusercontent.com/37813203/195024745-6a230eaf-6c78-474a-9215-684856ff8acf.png)

Reply:
done